### DISCUSSION TITLE
Turn on postUpdateOptions gomodTidy by default?

### DISCUSSION THREAD

**Discussion by danijam** [renovatebot/renovate#29324] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps v37.350.0

### Please tell us more about your question or problem

I'd like to globally configure renovate to use the go mod tidy post update options. This is off by default. In reading the docs it appears this is set on a per repository basis. I'd like to make this globally on rather than require each repo to configure the setting. Can I put this somewhere in my `config.json` on the bot admin side?

https://docs.renovatebot.com/modules/manager/gomod/#post-update-options

**Comment by HonkingGoose** [comment, marked as answer]
I think you're looking for a "local preset". That means that you have a file on the same platform (Azure DevOps) and refer to that file in each repository's Renovate config file.

But you must still put the reference in each of your repository's config files... So you might as well setup the go mod tidy stuff... 🙃

One benefit of the preset is that any _further_ things you want to customize globally are easier.

Read the [Renovate docs, config presets](https://docs.renovatebot.com/config-presets/) to learn more.

Maybe the maintainers have a better idea.
